doc/getting-started.md

```diff
@@ -38,6 +38,16 @@ $ echo SECRET_KEY=$(openssl rand -hex 32) >> .env
 
 This `SECRET_KEY` environment variable is currently the only required one.
 
+#### Set ALGORITHM and ACCESS_TOKEN_EXPIRE_MINUTES in environment file
+
+We need to specify an algorithm for JWT token encoding and decoding. ALGORITHM variable needs to be passed in the parameter for that.
+ALGORITHM is set default to HS256.
+We have used ACCESS_TOKEN_EXPIRE_MINUTES variable to set expiry time on generated jwt access token.
+ACCESS_TOKEN_EXPIRE_MINUTES is set default to None.
+If a user wants to change any of the above variables, they should be added to the .env file.
+
+Please find the [env.sample](https://github.com/kernelci/kernelci-api/blob/main/env.sample) file for the reference.
+
```

Comment:
This is not actually required to get started, which is why I didn't include it here.  It's something that could be added in the full API documentation under a section about environment variables.

Reply:
Okay, I'll add it to `API details` page.